chatterbot/storage/sqlalchemy_storage.py

```diff
@@ -98,30 +99,19 @@ def __init__(self, **kwargs):
             "read_only", False
         )
 
-        self.drop_create = self.kwargs.get(
-            "drop_create", False
-        )
+        create = self.kwargs.get("create", False)
 
-        if not self.read_only and self.drop_create:
-            _base.metadata.drop_all(self.engine)
-            _base.metadata.create_all(self.engine)
+        if not self.read_only and create:
+            self.create()
+
+        Session = sessionmaker(bind=self.engine)
+        self.session = Session()
```

Comment:
@davizucon / @navyad I removed the `__get_session()` method and created a single instance of the session object. The reason I did this was because I think it was creating a new session each time `__get_session` was called (and not disposing of it).

I'm not very familiar with SQLAlchemy. If either of you have any insight on this or any of the other changes please let me know.